packages/cli/src/index.ts

```diff
@@ -5,6 +5,7 @@ import { login, whoami } from './auth';
 import { aws } from './aws';
 import { bot, createBotDeprecate, deployBotDeprecate, saveBotDeprecate } from './bots';
 import { project } from './projects';
+import { MEDPLUM_VERSION } from '@medplum/core';
```

Comment:
There is a sonar warning regarding multiple imports of '@medplum/code`.

https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=1915&id=medplum_medplum&open=AYe5bOm_LR-8VPjwL99h

Reply:
Ah, this is a good catch.  The VS Code "Organize Imports" feature is great.  I recommend setting it up to run it on save (see https://stackoverflow.com/questions/49919726/how-to-invoke-the-organize-imports-typescript-feature-in-visual-studio) or run compulsively manually. 

<img width="606" alt="image" src="https://user-images.githubusercontent.com/749094/234376890-cf395578-0519-416d-adc8-bbc82409dbe0.png">